### PR TITLE
WIP: Test updating container networking

### PR DESCRIPTION
**Pull request by spowelljr**
Checking to see if this resolves the hairpinning issue we're encountering with the bridge CNI on HyperKit with k8s v1.24

### CODE REVIEW

**Comment by spowelljr** [comment]
ok-to-build-iso

**Comment by sharifelgamal** [comment]
```
RROR: No hash found for cni-plugins-linux-amd64-v1.1.1.tgz
make[2]: *** [/home/jenkins/go/src/k8s.io/minikube/out/buildroot/output-x86_64/build/cni-plugins-v1.1.1/.stamp_downloaded] Error 1
package/pkg-generic.mk:144: recipe for target '/home/jenkins/go/src/k8s.io/minikube/out/buildroot/output-x86_64/build/cni-plugins-v1.1.1/.stamp_downloaded' failed
```

just need to add a hash entry for cni-plugins v1.1.1 (and cni v1.1.1). Also, you should update it for aarch64 as well as x86_64.

File: deploy/iso/minikube-iso/arch/x86_64/package/cni-plugins/cni-plugins.hash [hunk 1]
```diff
@@ -3,3 +3,5 @@ sha256 8589670f7f9b211a351dfcd211d4fe0b961d77283a7415443dc188f3dbf05668  v0.6.0.
 sha256 92c7599918be0a720ac020f137cdeac746dfa03da6b26e08a37132c5728c091f  v0.7.5.tar.gz
 sha256 9d1526ed965ac6562fd95a931ab2346b3c5efd58c2f569038ba3c530f7e66472  v0.8.5.tar.gz
 sha256 bd682ffcf701e8f83283cdff7281aad0c83b02a56084d6e601216210732833f9  cni-plugins-linux-amd64-v0.8.5.tgz
+sha256 c86c44877c47f69cd23611e22029ab26b613f620195b76b3ec20f589367a7962  v1.1.1.tar.gz
```
Comment (sharifelgamal) [inline deploy/iso/minikube-iso/arch/x86_64/package/cni-plugins/cni-plugins.hash:6]: v1.1.1 should go in cni.hash

Comment (spowelljr) [reply deploy/iso/minikube-iso/arch/x86_64/package/cni-plugins/cni-plugins.hash:6]: That hashes file has two hashes for each version the linux-amd64 hash and the source code hash, so I added both as well. But I did forget to update the CNI hash which I have resolved. Do you know if both hashes are needed in the first file?

Comment (sharifelgamal) [reply deploy/iso/minikube-iso/arch/x86_64/package/cni-plugins/cni-plugins.hash:6]: it shouldn't be needed, i switch it from building from source, to just grabbing the binaries, which is why the format changed. you should just need the `cni-plugins-linux` formatted one

**Comment by spowelljr** [comment]
ok-to-build-iso

File: deploy/iso/minikube-iso/arch/x86_64/package/cni-plugins/cni-plugins.hash (hunk 1 above)
Comment (spowelljr) [reply deploy/iso/minikube-iso/arch/x86_64/package/cni-plugins/cni-plugins.hash]: Thanks for the clarification, removed the source code hash

**Comment by spowelljr** [comment]
```
>>> linux-headers 4.19.235 Downloading
--2022-06-06 17:51:52--  https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.19.235.tar.xz
Resolving cdn.kernel.org (cdn.kernel.org)... 151.101.193.176, 151.101.129.176, 151.101.65.176, ...
Connecting to cdn.kernel.org (cdn.kernel.org)|151.101.193.176|:443... connected.
HTTP request sent, awaiting response... 503 Backend unavailable, connection timeout
2022-06-06 17:51:54 ERROR 503: Backend unavailable, connection timeout.
```
I'm going to assume this is a flake and going to run again

**Comment by spowelljr** [comment]
ok-to-build-iso

**Comment by spowelljr** [comment]
```
3:02:05 /usr/bin/install -D -m 0755 /home/jenkins/go/src/k8s.io/minikube/out/buildroot/output-x86_64/build/cni-plugins-v1.1.1/flannel /home/jenkins/go/src/k8s.io/minikube/out/buildroot/output-x86_64/target/opt/cni/bin/flannel
13:02:05 /usr/bin/install: cannot stat '/home/jenkins/go/src/k8s.io/minikube/out/buildroot/output-x86_64/build/cni-plugins-v1.1.1/flannel': No such file or directory
13:02:05 package/pkg-generic.mk:339: recipe for target '/home/jenkins/go/src/k8s.io/minikube/out/buildroot/output-x86_64/build/cni-plugins-v1.1.1/.stamp_target_installed' failed
13:02:05 make[2]: *** [/home/jenkins/go/src/k8s.io/minikube/out/buildroot/output-x86_64/build/cni-plugins-v1.1.1/.stamp_target_installed] Error 1
13:02:05 make[1]: *** [_all] Error 2
13:02:06 Makefile:84: recipe for target '_all' failed
```
`flannel` graduated to its own repo since last update

**Comment by sharifelgamal** [comment]
so either we can downgrade, or we can give flannel its own package directory and remove the installation from cni-plugins

**Comment by spowelljr** [comment]
This is not causing the Hairpinning issue so going to close for now and can revisit in the future after the release

**Pull request closed without merging**